libvirt.go

```diff
@@ -510,6 +530,16 @@ func (d *Driver) getIPByMacFromSettings(mac string) (string, error) {
 		log.Warnf("Failed to get network bridge: %s", err)
 		return "", err
 	}
+
+	return bridge_name, nil
+}
+
+func (d *Driver) getIPByMacFromSettings(mac string) (string, error) {
+	bridge_name, err := d.getBridgeName()
+	if err != nil {
+		log.Warnf("Failed to get network bridge: %s", err)
+		return "", err
+	}
 	statusFile := fmt.Sprintf(dnsmasqStatus, bridge_name)
 	data, err := ioutil.ReadFile(statusFile)
 	type Lease struct {
```

Comment:
I don't think we should parse this file, using virDomainInterfaceAddresses would be better, but I'm not sure how well it works with qemu:///session. But that's preexisting, just a note for later.